docs/en/device/npu.md

```diff
@@ -2,7 +2,7 @@
 
 ## Usage
 
-Please refer to [link](https://github.com/open-mmlab/mmcv/blob/master/docs/zh_cn/get_started/build.md) installing mmcv on NPU Devices.
+Please refer to the [building documentation of MMCV](https://github.com/open-mmlab/mmdetection/pull/9535) to install MMCV on NPU devices
```

Comment:
https://github.com/open-mmlab/mmdetection/pull/9535?  The address you can assume has been merged.